src/hidimstat/permutation_feature_importance.py

```diff
@@ -1,57 +1,63 @@
 import numpy as np
 from sklearn.metrics import root_mean_squared_error
+from sklearn.model_selection import KFold
 from sklearn.utils import check_random_state
 
 from hidimstat.base_perturbation import BasePerturbation
+from hidimstat._utils.docstring import _aggregate_docstring
 
 
 class PFI(BasePerturbation):
+    """
+    Permutation Feature Importance algorithm
+
+    This as presented in :footcite:t:`breimanRandomForests2001`.
+    For each variable/group of variables, the importance is computed as
+    the difference between the loss of the initial model and the loss of
+    the model with the variable/group permuted.
+    The method was also used in :footcite:t:`mi2021permutation`
+
+    Parameters
+    ----------
+    estimator : sklearn compatible estimator, optionals
+        The estimator to use for the prediction.
+    method : str, default="predict"
```

Comment:
```suggestion
    prediction_method : str, default="predict"
```

Reply:
I use `method` everywhere; this is not the place to change it. If you want to change it, you should open an issue.